source/index.md

```diff
@@ -511,6 +511,9 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
   </dd>
 </dl>
 
+<p class="alert alert-info">
+<b>Notice:</b> A capture cannot be made after 180 days after the authorization.
```

Comment:
Ok. The "after" and "after" are valid, but my suggestion is also valid :-) It's up to you. I suggest similar alterations to refund and void.